.drone.yml

```diff
@@ -117,9 +135,34 @@ pipeline:
       matrix:
         TEST_SUITE: integration
 
+  lint-test:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - find . -name \*.php -not -path './vendor/*' -exec php -l "{}" \;
+    when:
+      matrix:
+        TEST_SUITE: lint
+
+  phpunit-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud
+      - chmod +x lib/composer/phpunit/phpunit/phpunit
```

Comment:
what is this? we better fix the release tar ball generation before spreading this workaround trough out to all repos.

Reply:
got the idea from you ;-)
https://github.com/owncloud/administration/pull/124/commits/1e7a058d66f21dbe4a83da8f75e9d7b5eca996f1

Reply:
we could use `php lib/composer/bin/phpunit`? no need to make it executeable and copy it around?